### DISCUSSION TITLE
How to delete item by Keyboard Bindings

### DISCUSSION THREAD

**Discussion by mingchi21** [lukasbach/react-complex-tree#29] [Q&A] [answered]
Can I delete selected item by Keyboard Bindings?

**Comment by lukasbach** [comment, marked as answer]
No, the lifecycle of items, such as item deletion, is not handled by rct, but you can do that yourself from the outside, e.g. by providing your data in a custom data provider and notifying subscribers with `onChangeItemChildren` that an item was removed on parents, or by using an `ControlledTreeEnvironment` and managing the complete item state yourself.
